smartcar-auth/src/main/java/com/smartcar/sdk/Helper.java

```diff
@@ -65,4 +65,14 @@ protected static String arrayToString(String[] array) {
         }
         return stringBuilder.toString();
     }
+
+    protected static VehicleInfo makeFullVehicle(String vin, String make, String model, Integer year) {
```

Comment:
I think we should be using `int` here and not the `Integer` class ([source](https://stackoverflow.com/a/10623753/9045128)).

Reply:
discussed IRL

Reply:
I was initially confused as to why we need the builder pattern too as it's not a really complex class, I just refactored the old authVehicleInfo class which used builder and added my new fields, I am thinking since we are making a breaking change anyway, now I can go and not use builder.

Reply:
There are some interesting answers here: https://stackoverflow.com/questions/328496/when-would-you-use-the-builder-pattern